Trappist/src/Promact.Trappist.Web/wwwroot/app/tests/shared/test-link-popover/test-link-popover.component.ts

```diff
@@ -0,0 +1,33 @@
+﻿import { Component, ElementRef, HostListener } from '@angular/core';
+
+@Component({
+    moduleId: module.id,
+    selector: 'test-link-popover',
+    templateUrl: 'test-link-popover.html'
+})
+export class TestLinkPopoverComponent {
+    public elementRef:any;
+
+    constructor(myElement: ElementRef) {
+        this.elementRef = myElement;
+    }
+
+    @HostListener('document:click', ['$event'])
```

Comment:
Opening & Closing of the popover would be handled by the developer and so after that this issue would also be handled.